stdlib/2and3/pstats.pyi

```diff
@@ -0,0 +1,36 @@
+from profile import Profile
+from cProfile import Profile as cProfile
+from typing import Any, AnyStr, Dict, IO, List, Tuple, TypeVar, Union, overload
+
+_Selector = Union[AnyStr, float, int]
+_T = TypeVar('_T', bound='Stats')
+
+class Stats:
+    # As best I can tell, the first argument has to be a string or Profile
+    # instance and the remaining ones may also be Stats instances
+    # the only keyword argument accepted is "stream"
+    def __init__(self: _T, *args: Union[AnyStr, Profile, _T],
+                 **kwds: IO[Any]) -> None: ...
+    def init(self, arg: Union[AnyStr, Profile, cProfile]) -> None: ...
+    def load_stats(self, arg: Union[AnyStr, Profile, cProfile]) -> None: ...
+    def get_top_level_stats(self) -> None: ...
+    def add(self: _T, *arg_list: Union[AnyStr, Profile, cProfile, _T]) -> _T: ...
+    def dump_stats(self, filename: str) -> None: ...
```

Comment:
`filename` can be `unicode` in Python 2. You can use `Union[bytes, typing.Text]` (or `AnyStr`).